scripts/get-cdnjs-list.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import json
+import subprocess
+
+links = {}
+data = subprocess.check_output(['links', '-dump', 'https://cdnjs.com/'])
```

Comment:
There should be some config with an array of which libraries to include. Hard-coding a few into the default themes is not really flexible.